higlass/utils.py

```diff
@@ -66,6 +66,11 @@ def recommend_datatype(filetype):
         "extensions": [".htime"],
         "datatypes": ["time-interval"],
     },
+    "bam": {
```

Comment:
Is this intended to be added? If so, how about adding a brief note so we know this PR also added something related to bam?